models/generated/workloads/public_ip.go

```diff
@@ -0,0 +1,34 @@
+package workloads
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+)
+
+var _ Workloader = (*PublicIP)(nil)
+var _ Capaciter = (*PublicIP)(nil)
+
+// PublicIP is a struct that defines the workload to reserve a public ip on the grid
+type PublicIP struct {
+	ReservationInfo `bson:",inline"`
+
+	IP            net.IP `bson:"ip" json:"ip"`
+	DestinationIP net.IP `bson:"destination_ip" json:"destination_ip"`
```

Comment:
We drop DestinationIP, not needed anymore

Reply:
Why?